src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -599,10 +621,12 @@ class VAMap extends Component {
     const coords = this.props.currentQuery.position;
     const position = [coords.latitude, coords.longitude];
     const facilityLocatorMarkers = this.renderFacilityMarkers();
-    const externalLink =
-      currentQuery.facilityType === LocationType.CC_PROVIDER
-        ? urgentCareLink
-        : otherToolsLink;
+    const showDialogUrgCare =
+      currentQuery.facilityType === LocationType.URGENT_CARE_FARMACIES ||
+      (currentQuery.facilityType === LocationType.URGENT_CARE &&
+        currentQuery.serviceType === 'NonVAUrgentCare')
+        ? urgentCareDialogLink
```

Comment:
is there a simpler way to express this?

Reply:
I don't think on this if statement, maybe in function with a switch for facility and then an if for service type, but I still think ternary makes it more readable and the way people write js now days